src/dcos_e2e/backends/_aws/__init__.py

```diff
@@ -0,0 +1,342 @@
+"""
+Helpers for creating and interacting with clusters on AWS.
+"""
+
+import uuid
+from ipaddress import IPv4Address
+from pathlib import Path
+from shutil import rmtree
+from tempfile import gettempdir
+from typing import Optional  # noqa: F401
+from typing import Any, Dict, Set, Type
+
+from dcos_e2e._vendor.dcos_launch import config, get_launcher
+from dcos_e2e._vendor.dcos_launch.aws import DcosCloudformationLauncher
+from dcos_e2e._vendor.dcos_launch.onprem import AbstractOnpremLauncher
+from dcos_e2e._vendor.dcos_launch.util import AbstractLauncher  # noqa: F401
+from dcos_e2e.backends._base_classes import ClusterBackend, ClusterManager
+from dcos_e2e.distributions import Distribution
+from dcos_e2e.node import Node
+
+
+class AWS(ClusterBackend):
+    """
+    A record of an AWS backend which can be used to create clusters.
+    """
+
+    def __init__(
+        self,
+        aws_region: str = 'us-west-2',
```

Comment:
This does not match what we recommend for cost - why?

If the reason for this region is cost, can you add a comment?